src/index.html

```diff
@@ -7,8 +7,53 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect" 
+      href="https://fonts.gstatic.com" 
+      value="crossorigin"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home" class="logo">
+        <img src="images/logo.png" alt="logo">
+      </a>
+      <nav class="nav">
+        <ul class="nav-ul">
+          <li class="nav-item">
```

Comment:
```suggestion
        <ul class="nav__ul">
          <li class="nav__item">
```
are these a separate blocks?